Author: Andreas Tille <tille@debian.org> modified for bioconda by Jon Puritz <jpuritz@gmail.com>
Description: Add pkg-config file to simplify building of freebayes
Forwarded: not-needed
--- /dev/null
+++ a/libtabixpp.pc
@@ -0,0 +1,12 @@
+prefix=${PREFIX}
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include/tabixpp
+
+Name: libtabixpp
+Version: 1.0
+Requires: 
+Description: C++ wrapper for tabix
+Libs: -L${libdir} -ltabixpp
+Cflags: -I${includedir}
+
